contrib/infiniband/ibvidentifier.h

```diff
@@ -1,26 +1,22 @@
 #ifndef IBVID_H
 #define IBVID_H
-#include <sys/types.h>
-#include <infiniband/verbs.h>
-#include <limits.h>
-#include <unistd.h>
-#include <stdlib.h>
+#include <stdint.h>
 
-struct ibv_qp_id {
+typedef struct ibv_qp_id {
   uint32_t qpn;
   uint16_t lid;
   uint32_t psn;
-};
+} ibv_qp_id;
 
-struct ibv_qp_pd_id {
+typedef struct {
   uint32_t qpn;
   uint32_t lid;
-};
+} ibv_qp_pd_id, ibv_ud_qp_id;
```

Comment:
Similarly, let's change new typedef types from `ibv_qp_pd_id` and `ibv_ud_qp_id` to `ibv_qp_pd_id_t`and `ibv_ud_qp_id _t`everywhere.